### DISCUSSION TITLE
Ladies and gentlemen, greetings. I am new to GitHub and I have a problem: I have a .hpw file that I cannot modify.

### DISCUSSION THREAD

**Discussion by Andrzej03** [community/community#137188] [Programming Help]
### Select Topic Area

Question

### Body

I have a .hpw file that is a language pack for configuring the web interface of a server. I would like to localize it into my language by modifying the existing file. However, the problem is that the file can be opened with WinRAR and contains several JavaScript files, but they cannot be edited. Do you have any advice on this?

**Comment by ebndev** [comment]
Hi @Andrzej03, thanks for being a part of the GitHub Community!

I will be removing the file from your post as we don't allow attachments.

You can check out [Lokalise](https://lokalise.com/?utm_campaign=CommunityInf_cm&utm_medium=social&utm_source=github) and see if they have documentation on how to do this! We'll be collaborating with them in the upcoming weeks as well if you have further questions about localization 😄
